test/cli/deps/error/index.js

```diff
@@ -0,0 +1,11 @@
+import 'node:path'
+
+export function testStack() {
```

Comment:
Currently ssr transform doesn't preserve lines with `export`, so this repro fails on main.